server/routerlicious/packages/services-utils/src/redisClientConnectionManager.ts

```diff
@@ -107,18 +121,38 @@ export class RedisClientConnectionManager implements IRedisClientConnectionManag
 			this.redisOptions.clusterRetryStrategy = this.redisOptions.retryStrategy;
 		}
 
+		const redisClusteringOptions: Partial<Redis.ClusterOptions> = {
+			redisOptions: this.redisOptions,
```

Comment:
This contains the Password, so I don't think we can log this as-is

Reply:
The "loggableClusteringOptions" object is the one being logged here, which has the password redacted:

`		// Remove password from the options objects that will be logged
		const loggableRedisOptions = { ...this.redisOptions };
		loggableRedisOptions.password = undefined;

		const loggableClusteringOptions = { ...redisClusteringOptions };
		loggableClusteringOptions.redisOptions = loggableRedisOptions;
`

It removes the password from loggableRedisOptions, then replaces loggableClusteringOptions's redisOptions field with the loggable(password removed) version

Reply:
🤦 totally missed that.